runtime/test-runtime/src/lib.rs

```diff
@@ -532,6 +532,7 @@ construct_runtime! {
 		ParasInherent: parachains_paras_inherent::{Pallet, Call, Storage, Inherent},
 		Initializer: parachains_initializer::{Pallet, Call, Storage},
 		Paras: parachains_paras::{Pallet, Call, Storage, Origin, Event},
+		ParasShared: parachains_shared::{Pallet, Call, Storage},
 		Scheduler: parachains_scheduler::{Pallet, Call, Storage},
 		ParasSudoWrapper: paras_sudo_wrapper::{Pallet, Call},
 		SessionInfo: parachains_session_info::{Pallet, Call, Storage},
```

Comment:
pallet must be put in `construct_runtime` for tests... if you make a call into `ParasShared` storage without doing so frame_support will panic as it needs a pallet name (the one that was previously declared in `decl_storage!`. Probably should check if any of cumulus tests do this